platform/barefoot/bfn-sai.mk

```diff
@@ -6,3 +6,4 @@ $(BFN_SAI)_RDEPENDS += $(LIBNL_GENL3)
 
 SONIC_ONLINE_DEBS += $(BFN_SAI)
 $(BFN_SAI_DEV)_DEPENDS += $(BFN_SAI)
+$(BFN_SAI_DEV)_CONFLICTS += $(LIBSAIVS_DEV)
```

Comment:
CONFLICTS is introduced to uninstall certain packages. BFN_SAI contains same sai header files as LIBSAIVS_DEV. So we need to install LIBSAIVS_DEV before install BFN_SAI